dluhc-component-library/src/components/siteSelectionForm/SiteSelectionForm.tsx

```diff
@@ -186,6 +196,7 @@ const SiteSelectionForm = ({ filepath, data, uiSchema }: SiteSelectionForm) => {
 
   const handleSubmitClicked = () => {
     uploadFile("local_plan", formData);
+    clearFormData();
```

Comment:
`Non-blocker`: not a huge issue for alpha, but we could do `uploadFile(...).then(() => clearFormData())` to make sure it only clears the data when uploading is successful

Reply:
➕